src/syntax_highlight.rs

```diff
@@ -34,7 +36,8 @@ unsafe impl Sync for Setup {}
 lazy_static!{
     static ref SETUP: Setup = Setup {
         syntax_set: SyntaxSet::load_defaults_newlines(),
-        theme_set: ThemeSet::load_defaults()
+        theme_set: ThemeSet::load_defaults(),
+        theme_name: RefCell::new(DEFAULT_THEME_NAME.to_owned()),
```

Comment:
Sounds like we should go with the alternative then :)

Thanks for all your work on this!